### ISSUE TITLE
Proxy configuration 

### ISSUE THREAD

**Issue by bucioo** [Aiven-Open/http-connector-for-apache-kafka#208]
Hi,
i have a use case that sink connector would suffise if it had Proxy configuration. Currently we cannot use it as not able to connect to SaaS product without proxy usage and we tried to use environment variables/add to JVM for proxy but that also didn't work. Is it possible to add it?
Regards

**Comment by jeqo** [comment]
Thanks @bucioo ! I think this make sense and can unblock other users having similar constraints. 

I've look into the Jdk http library and setting proxy host:port should do the trick.

Would you mind giving a try to #215 ? We may need to add some integration test to validate the approach, but let us know if it solves the issue for your environment.

**Comment by bucioo** [comment]
Hi @jeqo, thank you for adding proxy configuration. 

I was able to build application, not without problems as in our org we cannot download pkgs from internet, but i can confirm that error "java.nio.channels.UnresolvedAddressException" which i had before is no longer there.

Unfortunately we cannot connect to the endpoint via https as now i have error about certificates, same as #216. Tried to add key/trustore to JVM but no luck.
Pls suggest if there is a way to skip certs validation or this conf would need to be added to the code?
